frontend/packages/core/src/AppLayout/drawer.tsx

```diff
@@ -1,198 +1,213 @@
+const LinkHeading = styled(Typography)({
+  color: "rgba(13, 16, 48, 0.6)",
+  fontWeight: 500,
+  fontSize: "14px",
+  lineHeight: "18px",
+});
 
 interface GroupProps {
   heading: string;
   open: boolean;
   updateOpenGroup: (heading: string) => void;
-  onNavigate: () => void;
 }
 
-const Group: React.FC<GroupProps> = ({
-  heading,
-  open = false,
-  updateOpenGroup,
-  onNavigate,
-  children,
-}) => {
+const Group: React.FC<GroupProps> = ({ heading, open = false, updateOpenGroup, children }) => {
   const childrenList = React.Children.toArray(children);
 
+  const [openList, setListOpen] = React.useState(false);
+
+  const anchorRef = React.useRef(null);
+
+  const handleToggle = () => {
+    setListOpen(!openList);
+  };
+
+  const handleClose = event => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
+      return;
+    }
+    setListOpen(false);
+  };
+
+  let headingPath = window.location.pathname.replace("/", "").split("/")[0];
+
+  if (headingPath === "ec2") {
+    headingPath = "aws";
+  }
+
+  const isSelected = headingPath == heading.toLowerCase();
+
   // n.b. if a Workflow Grouping has no workflows in it don't display it even if
   // it's not explicitly marked as hidden.
   if (childrenList.length === 0) {
     return null;
   }
+
   return (
-    <List data-qa="workflowGroup">
-      <ListItem button onClick={() => updateOpenGroup(heading)}>
-        <GroupHeading color="primary" variant="h6">
-          {heading}
-        </GroupHeading>
-        <Grid container justify="flex-end" data-qa="toggle">
-          {open ? <ExpandLessIcon /> : <ExpandMoreIcon />}
-        </Grid>
-      </ListItem>
-      {!open ? <Divider variant="middle" /> : null}
-      <Collapse in={open} timeout="auto" unmountOnExit>
-        <List component="div" disablePadding>
-          {childrenList.map((c: React.ReactElement) => {
-            return React.cloneElement(c, { onClick: onNavigate });
-          })}
-        </List>
-      </Collapse>
-    </List>
+    <GroupList data-qa="workflowGroup">
+      <GroupListItem
+        button
+        selected={isSelected}
+        ref={anchorRef}
+        aria-controls={openList ? "workflow-options" : undefined}
+        aria-haspopup="true"
+        onClick={() => {
+          handleToggle();
+          updateOpenGroup(heading);
+        }}
+      >
+        <Avatar>{heading.charAt(0)}</Avatar>
+        <GroupHeading align="center">{heading}</GroupHeading>
+        <Collapse in={open} timeout="auto" unmountOnExit>
+          <Popper
+            open={openList}
+            anchorEl={anchorRef.current}
+            role={undefined}
+            transition
+            placement="right-start"
+          >
+            <Paper>
+              <ClickAwayListener onClickAway={handleClose}>
+                <List component="div" disablePadding id="workflow-options">
+                  {childrenList.map((c: React.ReactElement) => {
+                    return React.cloneElement(c);
```

Comment:
not clear on why this isn't just `{children}`

Reply:
hmm yeah it can just be `{children}`, thx for pointing it out. i also updated this https://github.com/lyft/clutch/blob/main/frontend/packages/core/src/AppLayout/drawer.tsx#L140 to be:
```
  if (children === 0) {
    return null;
  }
```
since `children` is a number.


@dschaller is this refactor fine?